pkg/my/domain.go

```diff
@@ -65,7 +70,7 @@ func Websites(client *node.Client, domainNames []string) ([]*models.Websites, er
 	}
 
 	for i := 0; i < len(domainNames); i++ { //nolint:varnamelen
-		contractAddress := string(contractAddresses[i].CandidateValue)
+		contractAddress := string(contractAddresses[i].CandidateValue[4:])
 
 		brokenChunks, err := getMissingChunkIds(client, contractAddress)
```

Comment:
I go we never use get to impelmetn getter. furthermore broken and missing are two different things...